src/modules/neutronsq/process.cpp

```
@@ -220,7 +225,7 @@ bool NeutronSQModule::process(Dissolve &dissolve, const ProcessPool &procPool)
     Filters::trim(repGR, referenceFTQMin_.value_or(0.0), ftQMax);
     auto rMin = weightedGR.total().xAxis().front();
     auto rMax = weightedGR.total().xAxis().back();
-    auto rho = rdfModule->effectiveDensity();
+    auto rho = *rdfModule->effectiveDensity();
```

Again, we should return an error if rho is unset.